test/unit/test-shadow-embed.js

```diff
@@ -316,6 +318,59 @@ describes.sandboxed('shadow-embed', {}, () => {
     });
   });
 
+  describe('installShadowStyle', () => {
+    let shadowRoot, shadowRoot2;
+
+    beforeEach(() => {
+      shadowRoot = document.createElement('div');
+      shadowRoot2 = document.createElement('div');
+      resetShadowStyleCacheForTesting(window);
+    });
+
+    afterEach(() => {
+      resetShadowStyleCacheForTesting(window);
```

Comment:
nit: shouldn't this only be before or after each test? why both

Reply:
Removed.